tests/analyzer/test_data_diagnosis.py

```diff
@@ -356,9 +360,57 @@ def test_mutli_rules(self):
         assert (
             details_row[1] == 'kernel-launch/wall_overhead(B/L: 0.0103 VAL: 0.0050 VAR: -51.27% Rule:lambda x:x<-0.5),'
             + 'mem-bw/D2H_Mem_BW(B/L: 24.3000 VAL: 10.0000 VAR: -58.85% Rule:lambda x:x<-0.5),' +
-            'rule3:lambda label:True if label["rule1"]+label["rule2"]>=2 else False'
+            'rule3:lambda label:True if label["rule1"]["violated"]+label["rule2"]["violated"]>=2 else False'
         )
 
+        # Test multi-rule using values of metrics in criteria lambda expression
+        diag1 = DataDiagnosis()
+        # test _run_diagnosis_rules_for_single_node
+        rules = {
+            'superbench': {
+                'rules': {
+                    'rule1': {
+                        'categories':
+                        'NCCL_DIS',
+                        'criteria':
+                        'lambda x:x==0',
+                        'function':
+                        'value',
+                        'store':
+                        True,
```

Comment:
Strange format

Reply:
this is by auto-format, I can not control it